tests/torch/data/configs/weekly/classification/cifar10/mobilenet_v2_cifar10_nas_SMALL.json

```diff
@@ -35,5 +35,10 @@
                 }
             }
         },
+        "search": {
```

Comment:
That config has just 3 epochs. Is it enough for reproducing results from paper? 
If it takes more than hour to reproduce results, we can add config to the sample directory and don't include it to validation

Reply:
It is not enough for reproducing the results of the paper. Let's include the config to reproduce the results and, as you mention, don't include it to validation. Thanks!

Reply:
Agree, it makes sense to not validate for a long time and add a reference config to the sample folder.
But at the same time, e2e nightly NAS test should be meaningful. If 3 epochs is enough to achieve good enough accuracy, it should be fine.